core/src/main/scala/io/qbeast/package.scala

```diff
@@ -0,0 +1,6 @@
+package io
+
+package object qbeast {
+  type IISeq[T] = scala.collection.immutable.Seq[T]
```

Comment:
Do we really need a pure alias for class from the standard library? I can understand if we substituted generic parameters or any other transformation which makes it more concrete, but shortening the package name...

Reply:
Probably not the best practice, but it improves the readability of the code. If Seq is used, then it looks like the one by default, which is mutable